spring/src/test/kotlin/de/codecentric/hikaku/converters/spring/deprecation/ConsumesTestController.kt

```diff
@@ -0,0 +1,33 @@
+package de.codecentric.hikaku.converters.spring.deprecation
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.*
+
+@SpringBootApplication
+open class DummyApp
+
+data class Todo(val description: String)
+
+@Controller
+open class NoDeprecationController {
+
+    @RequestMapping("/todos")
+    fun todos(@RequestBody todo: Todo) { }
+}
```

Comment:
I suggest that you use a leightweight example for all three controllers. That will put the focus back on the deprecation feature and you can get rid of 4 endpoints in each assertion in the **SpringConverterDeprecationTest.kt **

```
@Controller
open class NoDeprecationController {

    @GetMapping("/todos")
    fun todos(): Todo { }
}
```